### PR TITLE
Add GCN on CORA example notebook

### PR DESCRIPTION
**Pull request by adamnsch**
Based on #173

### CODE REVIEW

**Comment by adamnsch** [comment]
Probably easiest to review by checking out the branch and rendering the notebook :)

File: examples/import-sample-export-gnn.ipynb [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9169bf30",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Evaluate the trained GCN model on our test set\n",
+    "model.eval()\n",
```
Comment (breakanalysis) [inline examples/import-sample-export-gnn.ipynb:300]: does this also produce some metrics? i dont understand how that would work because the model doesnt know the data nor the test_mask right

**Review by breakanalysis — APPROVED** [review]
Very nice to have these examples!
Good work

File: examples/fastrp-and-knn.ipynb [hunk 2]
```diff
@@ -16,7 +16,7 @@
     "Running this notebook requires a localhost Neo4j server with a recent GDS version (2.0+) installed.\n",
     "We recommend using Neo4j Desktop.\n",
     "\n",
-    "Also required are the Python libraries neo4j ([Neo4j Python Driver](https://neo4j.com/docs/api/python-driver/current/index.html)) and of course the `graphdatascience` (see README for installation instructions)."
+    "Also required are course the `graphdatascience` Python library (see README for installation instructions)."
```
[the remark this answers is not in the document]
Comment (adamnsch) [reply examples/fastrp-and-knn.ipynb:19]: yes!

File: examples/import-sample-export-gnn.ipynb [hunk 3]
```diff
@@ -0,0 +1,350 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a575da60",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "G = gds.graph.load_cora()\n",
+    "\n",
+    "# Let's make sure we have exactly 2708 \"Paper\" nodes\n",
+    "print(G.node_labels())\n",
```
[the remark this answers is not in the document]
Comment (adamnsch) [reply examples/import-sample-export-gnn.ipynb:108]: I thought about that, but is it really the convention in notebooks? @FlorentinD @nvitucci wdyt?

[the remark this answers is not in the document]
Comment (nvitucci) [reply examples/import-sample-export-gnn.ipynb:108]: I think asserts are good for testing, but less "interesting" for a user. If the intention is to just show what is happening, I'd just leave the output cell there with the result.

[the remark this answers is not in the document]
Comment (FlorentinD) [reply examples/import-sample-export-gnn.ipynb:108]: I think for notebooks I would prefer the `print`. Dont know about conventions though.

I like the Nicolas approach of removing the comment but leaving the output

File: examples/import-sample-export-gnn.ipynb (hunk 1 above)
Comment (adamnsch) [reply examples/import-sample-export-gnn.ipynb:300]: https://stackoverflow.com/questions/60018578/what-does-model-eval-do-in-pytorch

Comment (adamnsch) [reply examples/import-sample-export-gnn.ipynb:300]: As is mentioned in the notebook that code is taken directly from the PyG example https://pytorch-geometric.readthedocs.io/en/latest/notes/introduction.html#learning-methods-on-graphs

File: examples/import-sample-export-gnn.ipynb (hunk 3 above)
[the remark this answers is not in the document]
Comment (breakanalysis) [reply examples/import-sample-export-gnn.ipynb:108]: the advantage of assert is that if this fails it will be easy to see where it failed instead of continuing with silent incorrect data.
I have seen assert in notebooks in MOOCs

File: examples/import-sample-export-gnn.ipynb (hunk 1 above)
Comment (breakanalysis) [reply examples/import-sample-export-gnn.ipynb:300]: thanks! that makes sense

File: examples/import-sample-export-gnn.ipynb (hunk 3 above)
[the remark this answers is not in the document]
Comment (adamnsch) [reply examples/import-sample-export-gnn.ipynb]: Agree that asserts could also be nice. And in fact, when we add testing to the notebook that might be the preferred way. But since there are mixed opinions we will leave it as is for now
